README.rst

```diff
@@ -4,6 +4,10 @@
    :target: https://travis-ci.org/nilearn/nilearn
    :alt: Build Status
 
+.. image:: https://ci.appveyor.com/api/projects/status/dn3b752wboo83taw?svg=true
+   :target: https://ci.appveyor.com/project/aabadie/nilearn
```

Comment:
To me, some permission is missing to make it fully functionnal : the nilean-ci account on appveyor should be added to the github nilearn team. But I might be wrong. Ideally, having a look at how things are done for joblib on the appveyor settings page would help a lot.